src/Ghcid.hs

```diff
@@ -181,10 +181,20 @@ data TermSize = TermSize
     ,termWrap :: WordWrap
     }
 
+-- | Modify IO to catch runtime exceptions.
+handleErrors :: IO () -> IO ()
+handleErrors = handle $ \err -> case err of
```

Comment:
This `case` seems redundant - why not just `\(UnexpectedExist cmd _ mmsg) ->`. FWIW, all the CI failures are because HLint things a LambdaCase would be better here - but actually the lambda case isn't required.